waveLib.py

```diff
@@ -441,8 +441,8 @@ def stats1D(fspec, frqbins, lowFreq=0.05, highFreq=0.5):
     df = np.diff(np.append(frqbins[0], frqbins), n=1)
 
     # truncating spectra to sea/swell band
-    # [idx, _] = sb.findbtw(frqbins, lowFreq, highFreq, type=3)
-    idx = np.argwhere((frqbins >= lowFreq) & (frqbins <= highFreq)).squeeze()
```

Comment:
please keep 

idx = np.argwhere((frqbins >= lowFreq) & (frqbins <= highFreq)).squeeze()

Reply:
Fixed.  these two lines should now be identical the deleted ones above.